optima/model.py

```diff
@@ -69,12 +69,18 @@ def model(simpars=None, settings=None, verbose=None, benchmark=False, die=True):
     # Defined for total (not by populations) and time dependent [npts]
     treatvs     = simpars['treatvs']     # viral suppression - ART initiators (P)
     if usecascade:
-        biofailure  = simpars['biofailure']  # biological treatment failure rate (P/T)
-        vlmonfr     = simpars['vlmonfr']     # Viral load monitoring frequency (N/T)
-        restarttreat = simpars['restarttreat']  # Time to restart ART (T)
+        biofailure    = simpars['biofailure']  # biological treatment failure rate (P/T)
+        vlmonfr       = simpars['vlmonfr']     # Viral load monitoring frequency (N/T)
```

Comment:
not a big deal, but i find this variable name very obtuse. how about freqvltest ?